app/models/form_profiles/va21686c.rb

```diff
@@ -1,11 +1,249 @@
 # frozen_string_literal: true
 
+class ScrubbedString < Virtus::Attribute
+  def coerce(value)
+    ['NONE'].include?(value.to_s.upcase) ? '' : value
+  end
+end
+
+module VA21686c
+  class FormAddress
+    include Virtus.model
+
+    attribute :address_type, ScrubbedString
+    attribute :street, ScrubbedString
+    attribute :street2, ScrubbedString
+    attribute :street3, ScrubbedString
+    attribute :city, ScrubbedString
+    attribute :state, ScrubbedString
+    attribute :country, ScrubbedString
+    attribute :postal_code, ScrubbedString
+    attribute :country, ScrubbedString
+    attribute :country_text, ScrubbedString
+    attribute :post_office, ScrubbedString
+    attribute :postal_type, ScrubbedString
+  end
+
+  class FormFullName
+    include Virtus.model
+
+    attribute :first, ScrubbedString
+    attribute :middle, ScrubbedString
+    attribute :last, ScrubbedString
+  end
+
+  class FormLocation
+    include Virtus.model
+
+    attribute :country_dropdown, ScrubbedString
+    attribute :country_text, ScrubbedString
+    attribute :city, ScrubbedString
+    attribute :state, ScrubbedString
+  end
+
+  class FormDependent
+    include Virtus.model
+
+    attribute :full_name, VA21686c::FormFullName
+    attribute :child_date_of_birth, ScrubbedString
+    attribute :child_in_household, Boolean
+    attribute :child_address, VA21686c::FormAddress
+    attribute :child_social_security_number, ScrubbedString
+    attribute :attending_college, Boolean
+    attribute :disabled, Boolean
+    attribute :married, Boolean
+    attribute :place_of_birth, VA21686c::FormLocation
+  end
+
+  class FormMarriage
+    include Virtus.model
+
+    attribute :date_of_marriage, ScrubbedString
+    attribute :location_of_marriage, VA21686c::FormLocation
+    attribute :spouse_full_name, VA21686c::FormFullName
+  end
+
+  class FormCurrentMarriage < FormMarriage
+    include Virtus.model
+
+    attribute :spouse_social_security_number, ScrubbedString
+    attribute :spouse_has_no_ssn, Boolean
```

Comment:
Should we prefill this for spouse and children, or just let the end user set these values via the form?
If 'yes', we should prefill `noSsnReasonType`.

Reply:
not sure, i would ask in the 686 channels if those values can be returned by the retrieve end point